2019-Q4.md

```diff
@@ -0,0 +1,166 @@
+# Ultimate goal: Enabling the world’s largest engineering organizations to have access to the data & insights they need to be successful.
+
+# Company OKRs
+
+
+## Company objective: Have a healthy financial status
+
+
+### Company key result: Close the funding round
+
+Initiative: Advance talks with funds/investors
+
+Initiative: Deliver a backup plan
+
+
+### Company key result: Reducing overhead (non-salaries) by minimum 20%
+
+Initiative: Deliver a study of expenses for the past 2 years
+
+Initiative: Deliver a plan of reducing less priorities costs
+
+
+### Company key result: Deliver a structured guideline for expenses
+
+Initiative: Create a report on the status of our current expenses guideline
+
+Initiative: Deliver a proposal for a new expenses guideline
+
+
+## Company objective: Go to market
+
+
+### Company key result: $1M in sales TCV
+
+Initiative: Deliver a campaign to leverage end of year budget to close faster 
+
+Initiative: Prioritize efforts on accounts already the proposal stage and develop incentives to shorten sales cycles
+
+Initiative:  Build a proper salesforce dashboard to visualize accounts & opportunities stages (shared with Data Intelligence)
+
+
+### Company key result: 3 clients PoC started
+
+Initiative: Hire solutions Engineer and further refine process to kick off PoC promptly 
+
+Initiative: Deliver Technical documentation & guides for Enterprise implementation and onboarding (i.e for different infrastructure, use cases, tech stacks)
+
+
+### Company key result: $3M new revenue in top of the funnel
+
+Initiative: Deliver more sales collaterals including top engineering metrics, Ebooks and datasheets 
+
+Initiative: Refocus Inbound lead generation efforts towards higher Enterprise profiles to increase % of requests from executives
+
+Initiative: Develop, execute and measure success of our Outbound efforts
+
+Initiative: Effectively support product releases with integrated campaigns 
+
+
+### Company key result: 3M revenue moved into proposal stage
+
+Initiative: Refine strategy and processes from MQA outreach to actual workshop delivery (shared with sales)
+
+Initiative: Plan and develop Enterprise workshop materials
+
+Initiative: Hire a pre-sales Engineer (shared with talent)
+
+Initiative: Develop pre-sales materials such RFP response, Datasheets, Objection Handling, the anatomy of a deal, demo scripts, Competitive teardowns & Technical roadmap (shared with sales)
+
+
+### Company key result: Getting user traction for source{d} CE
+
+Initiative: Setup the feedback loop with source{d{} users
+
+Initiative: Improve onboarding user experience 
+
+
+## Company objective: Deliver a unified product experience
+
+
+### Company key result: Bring query results under 10s for datasets of 10k repos
+
+Initiative: TBD
+
+Initiative: TBD
+
+
+### Company key result: Add 7 data sources
```

Comment:
Would be great to be more specific and prioritize the ones we know customers want.